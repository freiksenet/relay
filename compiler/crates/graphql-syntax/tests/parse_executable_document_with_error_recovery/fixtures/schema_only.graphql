scalar Foo

"""
Definition with comment
"""
type User {
  id: ID!
}