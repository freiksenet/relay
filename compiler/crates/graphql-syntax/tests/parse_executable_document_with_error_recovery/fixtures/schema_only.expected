==================================== INPUT ====================================
scalar Foo

"""
Definition with comment
"""
type User {
  id: ID!
}
==================================== OUTPUT ===================================
ExecutableDocument {
    span: 0:74,
    definitions: [],
}
Errors:
